README.md

```diff
@@ -284,6 +284,7 @@ The following resources can be specified within a `rule`:
 - `resources:`
   - `mem_mb = <INT>`: the memory required for the rule, in megabytes. If not specified,
     will [default to the amount you set when initialising](#default-mem-mb) the profile.
+- `time: <STR>`: the runtime limit required for the rule, in minutes. If not specified, will default to the amount you set when initialising the profile.
```

Comment:
Right and right. Pushed an update.
What happens if there is no time specified in the `lsf.yaml`? Maybe it's worth adding a default value to CookieCutter?

Reply:
If you think it would be needed then go ahead and add a default. Although, I think it would be best to leave the default blank and then if no default is present and the user hasn't set a runtime value we don't add `-W` to the submit command. This would be then default to the cluster default